src/nanoc/nanoc.yaml

```diff
@@ -1,6 +1,6 @@
-sbtVersion: 1.1.4
+sbtVersion: 1.1.5
 # Windows build version tends to be different from the rest
-windowsBuild: 1.1.4
+windowsBuild: 1.1.5
 
 sbtVersionForScalaDoc: 1.1.2
```

Comment:
is this an oversight?

Reply:
yes. we need a better oversight of these process :)

Reply:
`git grep 1\\.\\.` ftw